nyc_data/ppe/data_mappings.py

```diff
@@ -145,7 +145,6 @@ def to_objects(self):
             quantity=self.quantity,
             vendor=self.vendor,
             raw_data=self.raw_data,
-            data_source=DataType.EDC_PPE,
             order_type=OrderType.Purchase,
         )
```

Comment:
Should cost be a part of the `Purchase` object? https://github.com/RTCovid/nyc-ppe/blob/track-imports/nyc_data/ppe/models.py#L63

It's nullable, but am wondering if we want to add logic based on the individual row to see if we have `cost` available, and add it in, if so

Reply:
I think this spreadsheet just didn't have cost info